data/mods/Magiclysm/Spells/earthshaper.json

```diff
@@ -503,6 +503,160 @@
     "max_duration": 8640000,
     "duration_increment": 576000
   },
+  {
+    "type": "effect_on_condition",
+    "id": "EOC_EARTHSHAPER_UTILITY_MOLDING",
+    "effect": [
+      {
+        "run_eoc_selector": [
+          "EOC_EARTHSHAPER_UTILITY_MOLDING_HAMMER",
+          "EOC_EARTHSHAPER_UTILITY_MOLDING_PICKAXE",
+          "EOC_EARTHSHAPER_UTILITY_MOLDING_AXE",
+          "EOC_EARTHSHAPER_UTILITY_MOLDING_KNIFE",
+          "EOC_EARTHSHAPER_UTILITY_MOLDING_SHOVEL"
+        ],
+        "names": [ "Summon Hammer", "Summon Pickaxe", "Summon Axe", "Summon Knife", "Summon Shovel" ],
+        "keys": [ "1", "2", "3", "4", "5" ],
+        "descriptions": [
+          "Transform the living earth into a hammer.",
+          "Transform the living earth into a pickaxe.",
+          "Transform the living earth into an axe.",
+          "Transform the living earth into a knife.",
+          "Transform the living earth into a shovel."
+        ]
+      }
+    ]
+  },
+  {
+    "type": "effect_on_condition",
+    "id": "EOC_EARTHSHAPER_UTILITY_MOLDING_HAMMER",
+    "effect": [
+      { "u_remove_item_with": "eshaper_pickaxe" },
```

Comment:
Maybe not remove molding tools if you cast a spell? what if someone need axe and knife simultaneously?

Reply:
Was it previously possible to have multiple items?  I assumed it wasn't--if it was then you're right, there's no reason to remove existing ones

Reply:
Looks like the original does allow for multiple tools.  I'll take out the remove_items